app/javascript/src/components/product-card.vue

```diff
@@ -0,0 +1,65 @@
+<template>
```

Comment:
Definition for rule 'vue/max-len' was not found  vue/max-len

Reply:
El mono nos sigue hinchando con 'vue/max-len' después de mil PRs